resources/roca_lib.brs

```diff
@@ -0,0 +1,260 @@
+' Creates a suite of test cases with a given description.
+' @param description a string describing the suite of tests contained within.
+' @param func the function to execute as part of this suite.
+' @param
```

Comment:
@sjbarag you accidentally a comment!

Reply:
Undocumented params/properties reminds me of some other programming language...